compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/ParameterDesugar.java

```diff
@@ -516,7 +516,7 @@ private BLangExpression createClosureForDefaultValue(String closureName, String
                                                          BLangSimpleVariable varNode) {
         BSymbol owner = getOwner(env);
         BInvokableTypeSymbol symbol = (BInvokableTypeSymbol) env.node.getBType().tsymbol;
-        BLangFunction function = createFunction(closureName, varNode.pos, symbol.pkgID, owner, varNode.getBType());
+        BLangFunction function = createFunction(closureName, varNode.pos, owner.pkgID, owner, varNode.getBType());
```

Comment:
This change is. not required, but as a workaround for https://github.com/ballerina-platform/ballerina-lang/issues/38412, this change was done